web_poet/_base.py

```diff
@@ -0,0 +1,40 @@
+"""Contains some internal definitions that is internal to **web-poet**.
+
+In general, users shouldn't import and use the contents of this module.
+"""
+
+
+from typing import Type, TypeVar, List, Dict
+
+from multidict import CIMultiDict
+
+T_headers = TypeVar("T_headers", bound="_HttpHeaders")
+
+
+class _HttpBody(bytes):
```

Comment:
Not sure about this :) What's the use case? Unlike headers, request body seems to be quite different from response body.

Reply:
I was initially planning to house alternative constructors like in https://github.com/scrapinghub/web-poet/pull/36 inside it. But mulling over it now, it's not a big of a motivation to introduce `_HttpBody` that what I initially envisioned.

Thanks for raising this! Reverted in https://github.com/scrapinghub/web-poet/pull/22/commits/3310c1b5c04281765407d190f85c8cfe638ddeab.